extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -134,8 +134,8 @@ void process(BuildProducer<SubstrateSystemPropertyBuildItem> substrateProps,
         addReflectiveClass(false, false, "com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl");
         addReflectiveClass(true, false, "com.sun.xml.bind.v2.ContextFactory");
         addReflectiveClass(true, false, "com.sun.xml.internal.bind.v2.ContextFactory");
-        runtimeClasses
-                .produce(new RuntimeInitializedClassBuildItem("com.sun.xml.internal.bind.v2.runtime.reflect.opt.Injector"));
+        //        runtimeClasses
+        //                .produce(new RuntimeInitializedClassBuildItem("com.sun.xml.internal.bind.v2.runtime.reflect.opt.Injector"));
```

Comment:
Mmmmh?

Reply:
My understanding is that the JAXB `com.sun.xml.internal.bind.v2.runtime.reflect.opt.Injector` class is only used from `com.sun.xml.internal.bind.v2.runtime.reflect.opt.AccessorInjector#prepare` method, which is substituted to an empty method (see https://github.com/quarkusio/quarkus/blob/master/extensions/jaxb/runtime/src/main/java/io/quarkus/jaxb/runtime/graal/JAXBSubstitutions.java#L83).  The result is that there is no reference to this class anymore and GraalVM throws an exception when building compiling to native.
I suppose this may have been caused by a change in Graal.

Reply:
@gsmet I added another commit to remove the commented code

Reply:
@gnodet cool thanks. Sorry to be picky but could you amend the commit message with your explanation. That would be useful for history if one day we ask ourselves why we removed it.

Once it's done, I'll merge.

Reply:
@gsmet done

Reply:
this one is really important to make the camel extension works.